docs/usage/types/strict_types.md

```diff
@@ -57,3 +62,89 @@ m = Model(finite=1.0)
 print(m)
 #> finite=1.0
 ```
+
+### Arguments to `conbytes`
```

Comment:
I don't think it's a good idea to have these "Arguments" sections. It's pretty much the same as sharing the API documentation.

Reply:
They can easily become out-dated.

Reply:
Just a link to the API docs then? Do API docs have the same info on them?

Reply:
Yes, and yes (if not, they should).

Reply:
Yes the api docs have this, I'll revert to what we had previously